src/implot_visualization/app/deserialize_json.cpp

```diff
@@ -0,0 +1,81 @@
+#include <deserialize_json.h>
+#include <iostream>
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+extern ScrollingBuffer buffer;
```

Comment:
I think we should try to remove this dependency and don't work with a global buffer.